requirements.txt

```diff
@@ -4,3 +4,4 @@ itsdangerous==0.24
 Jinja2==2.10
 MarkupSafe==1.0
 Werkzeug==0.14.1
+Flask-OAuthlib == 0.9.5
```

Comment:
Did you type this manually?

Reply:
use `pip freeze > requirements.txt`

Reply:
Yes 😬
From now onwards will use this command.